packages/astro/src/core/errors/errors-data.ts

```diff
@@ -15,23 +15,57 @@ export const AstroErrorData = defineErrors({
 		code: 1000,
 	},
 	// 1xxx and 2xxx codes are reserved for compiler errors and warnings respectively
+	/**
+	 * @docs
+	 * @see
+	 * - [Enabling SSR in Your Project](https://docs.astro.build/en/guides/server-side-rendering/#enabling-ssr-in-your-project)
+	 * - [Astro.redirect](https://docs.astro.build/en/guides/server-side-rendering/#astroredirect)
+	 * @description
+	 * The `Astro.redirect` function is only available when [Server-side rendering](/en/guides/server-side-rendering/) is enabled.
+	 *
+	 * To redirect on a static website, the [meta refresh attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta) can be used. Certain hosts also provide config-based redirects (ex: [Netlify redirects](https://docs.netlify.com/routing/redirects/)).
+	 */
 	StaticRedirectNotAllowed: {
 		code: 3001,
 		message:
 			"Redirects are only available when using output: 'server'. Update your Astro config if you need SSR features.",
 		hint: 'See https://docs.astro.build/en/guides/server-side-rendering/#enabling-ssr-in-your-project for more information on how to enable SSR.',
 	},
-	SSRClientAddressNotAvailableInAdapter: {
```

Comment:
We don't normally allow changing names, but since the docs part doesn't exist yet, this is ok. In general I want to strive towards slightly shorter names, we'll do longer names another way